docs/tutorial.md

```diff
@@ -68,19 +68,21 @@ del *
 cd ..
 ```
 
-4. Add a file named `index.css` in the `src/` folder with [this CSS code](https://codepen.io/gaearon/pen/oWWQNa?editors=0100).
+4. Add a file named `styles.css` in the `src/` folder and copy the content from `src/styles.css` in [the tutorial codesandbox](https://codesandbox.io/s/formik-v2-tutorial-start-s04yr).
 
-5. Add a file named `index.js` in the `src/` folder with [this JS code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010).
+5. Add a file named `index.js` in the `src/` folder and copy the content from `src/index.js` in [the tutorial codesandbox](https://codesandbox.io/s/formik-v2-tutorial-start-s04yr).
 
-6. Add these three lines to the top of `index.js` in the `src/` folder:
+6. Add all dependencies used in tutorial with NPM/yarn:
 
-```js
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
+```bash
+npm install --save formik yup @emotion/core @emotion/styled
```

Comment:
just download the sandbox and npm install?

Reply:
Got it, thanks Jared. That was a derp on my part – I couldn't find the download functionality on codesandbox.

PR updated. Makes the instructions MUCH shorter!